### PR TITLE
Add column arn for the table alicloud_rds_instance. Closes #147

### LINKED ISSUE #147: Add column arn for the table alicloud_rds_instance
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
Required for the query in new format.

**Describe alternatives you've considered**
N/A

**Additional context**
https://docs.runpanther.io/resources/aws/ec2-network-acl

### PR DESCRIPTION
**Pull request by rajeshbal65**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
### Basic info

```sql
select
  db_instance_id,
  arn,
  vpc_id,
  creation_time,
  engine
from
  alicloud_rds_instance;
+----------------------+-----------------------------------------------------------------------+---------------------------+---------------------+------------+
| db_instance_id       | arn                                                                   | vpc_id                    | creation_time       | engine     |
+----------------------+-----------------------------------------------------------------------+---------------------------+---------------------+------------+
| pgm-6gj950v5gnc8bqar | arn:acs:rds:ap-south-1:5982111499156037:instance/pgm-6gj950v5gnc8bqar | vpc-a2duau0qa8opncvr4kzq6 | 2021-06-10 12:31:12 | PostgreSQL |
+----------------------+-----------------------------------------------------------------------+---------------------------+---------------------+------------+
```
```
</details>

### CODE REVIEW

File: alicloud/table_alicloud_rds_instance.go [hunk 1]
```diff
@@ -94,6 +101,20 @@ func tableAlicloudRdsInstance(ctx context.Context) *plugin.Table {
 				Transform:   transform.FromField("DBInstanceClass"),
 				Description: "The instance type of the instances.",
 			},
+			{
+				Name:        "sql_collector_status",
```
Comment (bigdatasourav) [inline alicloud/table_alicloud_rds_instance.go:105]: the column name will be audit_status

File: alicloud/table_alicloud_rds_instance.go [hunk 2]
```diff
@@ -94,6 +101,20 @@ func tableAlicloudRdsInstance(ctx context.Context) *plugin.Table {
...
+				Hydrate:     getSqlCollectorPolicy,
+				Transform:   transform.FromField("SQLCollectorStatus"),
+				Description: "The status of the SQL Explorer (SQL Audit) feature.",
+			},
+			{
+				Name:        "audit_retention_duration",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getSqlCollectorRetention,
+				Transform:   transform.FromField("ConfigValue"),
+				Description: "The log backup retention duration that is allowed by the SQL explorer feature on the instance.",
```
Comment (bigdatasourav) [inline alicloud/table_alicloud_rds_instance.go:116]: The audit retention duration that is allowed by the SQL explorer feature on the instance.

File: alicloud/table_alicloud_rds_instance.go [hunk 3]
```diff
@@ -546,7 +567,7 @@ func listRdsInstances(ctx context.Context, d *plugin.QueryData, _ *plugin.Hydrat
 			return nil, err
 		}
 		for _, i := range response.Items.DBInstance {
-			plugin.Logger(ctx).Warn("alicloud_rds.DescribeDBInstances", "item", i)
+			plugin.Logger(ctx).Warn("alicloud_rds.DescribeDBInstancesxxxxxxxxx", "item", i)
```
Comment (bigdatasourav) [inline alicloud/table_alicloud_rds_instance.go:570]: provide proper message

File: alicloud/table_alicloud_rds_instance.go [hunk 4]
```diff
@@ -647,7 +668,7 @@ func getTDEDetails(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateDa
 	request.DBInstanceId = id
 	response, err := client.DescribeDBInstanceTDE(request)
 	if serverErr, ok := err.(*errors.ServerError); ok {
-		if serverErr.ErrorCode() == "InvalidDBInstanceId.NotFound" {
+		if serverErr.ErrorCode() == "InstanceEngineType.NotSupport" {
```
Comment (bigdatasourav) [inline alicloud/table_alicloud_rds_instance.go:671]: should have InvalidDBInstanceId.NotFound or InvalidDBInstanceId.NotFound

File: alicloud/table_alicloud_rds_instance.go [hunk 5]
```diff
@@ -768,7 +789,57 @@ func getRdsInstanceAkas(ctx context.Context, d *plugin.QueryData, h *plugin.Hydr
 	}
 	commonColumnData := commonData.(*alicloudCommonColumnData)
 	accountID := commonColumnData.AccountID
-	return []string{"acs:rds:" + region + ":" + accountID + ":instance/" + instanceID}, nil
+	return "acs:rds:" + region + ":" + accountID + ":instance/" + instanceID, nil
```
Comment (bigdatasourav) [inline alicloud/table_alicloud_rds_instance.go:792]: check the arn format once - it should be -arn:acs:dds:cn-hangzhou:123456789xxxxxxx:instance/dds-uf6ab1exxxx

File: alicloud/table_alicloud_rds_instance.go [hunk 6]
```diff
@@ -768,7 +789,57 @@ func getRdsInstanceAkas(ctx context.Context, d *plugin.QueryData, h *plugin.Hydr
...
+		return nil, err
+	}
+
+	var id = databaseID(h.Item)
+
+	request := rds.CreateDescribeSQLCollectorPolicyRequest()
+	request.Scheme = "https"
+	request.DBInstanceId = id
+	request.RegionId = region
+	request.DBInstanceId = databaseID(h.Item)
```
Comment (bigdatasourav) [inline alicloud/table_alicloud_rds_instance.go:811]: DBInstanceId set twice

File: alicloud/table_alicloud_rds_instance.go [hunk 7]
```diff
@@ -768,7 +789,57 @@ func getRdsInstanceAkas(ctx context.Context, d *plugin.QueryData, h *plugin.Hydr
...
+		return nil, err
+	}
+
+	var id = databaseID(h.Item)
+
+	request := rds.CreateDescribeSQLCollectorRetentionRequest()
+	request.Scheme = "https"
+	request.DBInstanceId = id
+	request.RegionId = region
+	request.DBInstanceId = databaseID(h.Item)
```
Comment (bigdatasourav) [inline alicloud/table_alicloud_rds_instance.go:836]: DBInstanceId set twice

**Comment by bigdatasourav** [comment]
@rajeshbal65 plz resolve.

File: alicloud/table_alicloud_rds_instance.go [hunk 8]
```diff
@@ -768,7 +789,51 @@ func getRdsInstanceAkas(ctx context.Context, d *plugin.QueryData, h *plugin.Hydr
 	}
 	commonColumnData := commonData.(*alicloudCommonColumnData)
 	accountID := commonColumnData.AccountID
-	return []string{"acs:rds:" + region + ":" + accountID + ":instance/" + instanceID}, nil
+	return "acs:acs:rds:" + region + ":" + accountID + ":instance/" + instanceID, nil
```
Comment (Subhajit97) [inline alicloud/table_alicloud_rds_instance.go:792]:
```suggestion
	return "arn:acs:rds:" + region + ":" + accountID + ":instance/" + instanceID, nil
```

**Review by Subhajit97 — CHANGES_REQUESTED** [review]
@rajeshbal65 There is a small change required in `arn` format. Please see comments, thanks!

**Review by Subhajit97 — APPROVED** [review]
LGTM

**Review by bigdatasourav — APPROVED** [review]
LGTM

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**